x-pack/plugins/security_solution/public/app/home/template_wrapper/index.tsx

```diff
@@ -88,16 +93,18 @@ export const SecuritySolutionTemplateWrapper: React.FC<Omit<KibanaPageTemplatePr
           $isShowingTimelineOverlay={isShowingTimelineOverlay}
           paddingSize="none"
           solutionNav={solutionNavProps}
-          restrictWidth={showEmptyState ? NO_DATA_PAGE_MAX_WIDTH : false}
```

Comment:
Current host page renders landing page component when no data exists:
Restrict the width makes timeline displayed incorrectly:
<img width="2545" alt="Screenshot 2023-06-21 at 11 04 16" src="https://github.com/elastic/kibana/assets/6295984/d41cd809-12dd-40ef-bf93-773e42697ae9">